java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/AbstractObservableLocalReader.java

```diff
@@ -111,6 +115,14 @@
         TIME_OUT
     }
 
+    /* The observers of this object */
+    private Set<ObservableReader.ReaderObserver> observers;
```

Comment:
observers private attribute: the observers can't be managed though a 'Set', but only though a 'List'. Because the observers are unknown objects. A Set collection requires that the objects implements the methods 'equals': impossible if the objects are unknown.